lib/rubycritic/generators/global_rating_calculator.rb

```diff
@@ -0,0 +1,15 @@
+module Rubycritic
+  module Generator
+    class GlobalRatingCalculator
```

Comment:
You never plan to instantiate this, right?
Then let's rather use a module and `module_function` instead of self :)